tests/manual-test-cases/Group5-Functional-Tests/5-22-NFS-Volume.md

```diff
@@ -0,0 +1,83 @@
+Test 5-22 - NFS Volume
```

Comment:
@matthewavery Can probably answer this a bit further, but my understanding is that "volume support for VIC starts at deployment time when invoking vic-machine create."  So this would be where we would add the NFS volume stores that we want to use.  And for variations, at the moment we can only do read/write volumes and not read-only.  I can setup different configurations of the NFS server like different disk size and whether we want to use "share" or "/exports/mount-point" instead but for these tests the NFS configuration was not important.